### ISSUE TITLE
Support beam cannot be deleted

### ISSUE THREAD

**Issue by Charles7523** [anegostudios/VintageStory-Issues#3603]
### Game Version

v1.19.3

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

I currently have a support beam hanging in mid-air which I cannot delete.

### How to reproduce

1. I built four vertical fence posts, each 4 blocks high. I then connected the tops of them with horizontal support beams. 
2. I then used wood slabs to make a makeshift roof.
3. Unhappy with the look of the structure, I decided to delete it all.
4. Three of the horizontal support beams were successfully removed. The fourth, however, refuses to delete. I get no hitbox indicator when looking at the end of the beam.

### Screenshots

![Hanging Beam](https://github.com/anegostudios/VintageStory-Issues/assets/160371834/2ab81090-82ab-45f7-9411-effb347e07f6)

### Logs

```Log```

**Comment by Craluminum2413** [comment]
Closed due to inactivity

**Comment by Craluminum2413** [comment]
Does it still happen in 1.21.0-rc.1?

**Comment by rex3344** [comment]
> Does it still happen in 1.21.0-rc.1?

Yeah i am still currently getting the issue i just checked in to see if it has been reported yet
